src/scenes/Nuke/Nuke.tsx

```diff
@@ -1,20 +1,33 @@
-import { memo, useEffect } from 'react';
+import { memo, useCallback, useEffect } from 'react';
 import { Link, RouteComponentProps } from '@reach/router';
 import styled from 'styled-components';
 import { Heading } from 'components/core/Text/Text';
 import Button from 'components/core/Button/Button';
 import Spacer from 'components/core/Spacer/Spacer';
 import Page from 'components/core/Page/Page';
 import { useAuthActions } from 'contexts/auth/AuthContext';
-import useNuke from 'hooks/api/collections/nuke/useNuke';
+import useFetcher from 'contexts/swr/useFetcher';
 
 // suggest a user visit this page if they're in a seriously broken state
 function Nuke(_: RouteComponentProps) {
   const { logOut } = useAuthActions();
 
-  useNuke();
+  const fetcher = useFetcher();
 
-  useEffect(logOut, [logOut]);
+  const nukeDb = useCallback(async () => {
+    try {
+      // attempt to nuke database if endpoint is available.
+      // this will only work on dev!
+      await fetcher('/nuke', 'nuke database');
+    } catch (e) {
+      // error silently
+    }
+  }, [fetcher]);
+
+  useEffect(() => {
+    logOut();
+    nukeDb();
```

Comment:
Maybe we can check the env and only call this if dev, so we don't make the request at all on prod